k8s-deep-dive-charly-scratchpad/08b-logs-permissions.md

```diff
@@ -41,15 +41,21 @@ service account permissions are.<br/><br/>
 In this case you will need to add permissions for the `list` verb to the `/pods`
 resource.<br/><br/>
 
-We included a sample patch as a solution:<br/><br/>
+We included a sample patch as a solution. Run the following to use it:<br/><br/>
 `kubectl patch clusterroles pod-lister --patch="$(cat assets/workshop-assets/apps/fixes/rbac-fix.yaml)"`{{execute}}
 
-NB: If the pod is in a CrashloopBackoff State:
+Check if `pod-lister` Pod is still in now running:<br/><br/>
+`kubectl get pods`{{execute}}
+
+If it is still in a CrashloopBackoff State, such as:
 ```
 pod-lister-b754c75db-rsz9s                       0/1     CrashLoopBackOff   5          4m33s
 ```
 
-Feel free to delete it, the deployment controller will create a new pod using the new RBAC that will be in a running state.
+Feel free to delete it by running the following (update the pod name based on your environment):<br/><br/>
+`kubectl delete pod pod-lister-b754c75db-rsz9s`{{copy}} 
```

Comment:
Running the command below:
```
`kubectl delete po $(kubectl get pods -lapp=pod-lister -o custom-columns=:metadata.name)`{{execute}}
```

will do the trick

Reply:
thanks!